eos/b-decays/b-to-psd-l-nu.hh

```diff
@@ -58,7 +58,7 @@ namespace eos
             // Integrated Observables - normalized(|Vcb|=1)
             double normalized_integrated_branching_ratio(const double & s_min, const double & s_max) const;
 
-            // R_D
+            // R_P
             double differential_r_d(const double & s) const;
```

Comment:
We could rename this from "..._r_d" -> "..._r_P"

Reply:
Resolved using ObservableRatio. This code has been removed altogether...